go/client/cmd_device_remove.go

```diff
@@ -134,7 +138,11 @@ func NewCmdDeviceRemove(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 		Flags: []cli.Flag{
 			cli.BoolFlag{
 				Name:  "f, force",
-				Usage: "Override warning about removing the current device.",
+				Usage: "Force removal of the current device.",
+			},
+			cli.BoolFlag{
+				Name:  "last",
```

Comment:
My thought was that this is a drastic measure, so requiring them to specify the device name or ID is an extra confirmation that they know what they are doing.